<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesKeyTypeDialog</class>
 <widget class="QDialog" name="PropertiesKeyTypeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Properties Type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="m_exactly">
        <property name="text">
         <string>Properties for exactly this source in the given channel</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_exactlyButAnyChannel">
        <property name="text">
         <string>Properties for exactly this source in any channel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_typeAndChannelOnly">
        <property name="text">
         <string>Properties for sources of the given type in the given channel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_typeInAnyChannel">
        <property name="text">
         <string>Properties for the sources of the given type in any channel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttons</sender>
   <signal>rejected()</signal>
   <receiver>PropertiesKeyTypeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
